lab-james/src/main.js

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import ReactDom from 'react-dom';
+import './style/main.scss';
+import faker from 'faker';
+import {say, SQUIRREL, DEFAULT, VADER, SKELETON} from 'cowsay';
+
+class Header extends React.Component {
+
+  constructor(props){
+    super(props);
+  }
+
+  render(){
+
+    return (
+      <header>
+        <h1>What does the cow say?</h1>
+      </header>
+    );
+  }
+}
+
+class App extends React.Component {
+
+  constructor(props){
+    super(props);
+
+    this.handleCowSay = this.handleCowSay.bind(this);
+    this.updateStateSay = this.updateStateSay.bind(this);
+    this.updateStateCow = this.updateStateCow.bind(this);
+    this.chooseCowReg = this.chooseCowReg.bind(this);
+    this.chooseCowDarth = this.chooseCowDarth.bind(this);
+    this.chooseCowSkeleton = this.chooseCowSkeleton.bind(this);
+
+    this.state = {
+      content: 'Moo',
+      cow: DEFAULT
+    };
+
+  }
+
+  handleCowSay(e){
+    this.updateStateSay(this.state.content);
+  }
+
+  updateStateSay(content){
+    let comment = faker.fake('{{lorem.words}}');
+    this.setState({content: comment});
+  }
+
+  chooseCowReg(e){
+    this.updateStateCow(DEFAULT);
+  }
+
+  chooseCowDarth(e){
+    this.updateStateCow(VADER);
+  }
+
+  chooseCowSkeleton(e){
+    this.updateStateCow(SKELETON);
+  }
+
+  updateStateCow(choice){
```

Comment:
You can probably do this with one function rather than three functions that are essentially the same.